dictionaries/en_US/src/en_US.txt

```diff
@@ -22712,6 +22715,7 @@ whists
 whitecapper
 whitecapping
 whitecomb
+whitepsace
```

Comment:
`whitespace` is already in the US English dictionary.

The command `trace` command can be used to find words. `cspell trace whitespace`

<img width="777" alt="image" src="https://github.com/streetsidesoftware/cspell-dicts/assets/3740137/91159d57-4a5a-4f43-b737-c5ec3f094382">

Reply:
Yep, found it this way. Thanks for the `trace` 👍